lib/livebook/runtime/evaluator.ex

```diff
@@ -675,6 +675,85 @@ defmodule Livebook.Runtime.Evaluator do
   end
 
   defp eval(:erlang, code, binding, env) do
+    try do
+      is_module = String.starts_with?(code, "-module(")
+
+      case is_module do
+        true -> eval_module(:erlang, code, binding, env)
+        false -> eval_statements(:erlang, code, binding, env)
+      end
+    catch
+      kind, error ->
+        stacktrace = prune_stacktrace(:erl_eval, __STACKTRACE__)
+        {{:error, kind, error, stacktrace}, []}
```

Comment:
I think this outer try-catch is not necessary. :)

Reply:
Morning - this has been cleaned up in latest update.